src/components/Step1.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class='main'>
```

Comment:
html側では文字列は`""`で囲みましょう。
```suggestion
  <div class="main">
```